samples/subsys/mgmt/osdp/sample.yaml

```diff
@@ -0,0 +1,12 @@
+sample:
+  name: OSDP Sample
+tests:
+  sample.subsys.mgmt.osdp:
+    tags: osdp
+    filter: dt_compat_enabled_with_alias("gpio-leds", "led0")
+    harness: osdp
+    integration_platforms:
+      - frdm_k64f
+
+    # FIXME: exclude due to build error
+    platform_exclude: qemu_xtensa
```

Comment:
look at this

https://github.com/zephyrproject-rtos/zephyr/pull/27913#issuecomment-685153646

Reply:
cool, just remove that commit.

Reply:
Thanks! I'll focus on osdp/sample.yaml there (#27913). For Kconfig, the comment^^ mentioned to also remove `SERIAL_HAS_DRIVER` and `UART_INTERRUPT_DRIVEN`. Do we need that too?

Reply:
I haven't checked that to be honest. Will look